src/tools/genpolicy/src/policy.rs

```diff
@@ -410,8 +410,8 @@ impl AgentPolicy {
                 config: config.clone(),
             })
         } else {
-            panic!("Cannot open file {}. Please copy it to the current directory or specify the path to it using the -i parameter.",
```

Comment:
It seems that the `Please copy it to the current directory or specify the path to it using the -i parameter.` is still valid, expect that it is `-p`.